dGame/dUtilities/SlashCommands/GMGreaterThanZeroCommands.cpp

```diff
@@ -287,4 +287,29 @@ namespace GMGreaterThanZeroCommands {
 		std::string name = entity->GetCharacter()->GetName() + " - " + args;
 		GameMessages::SendSetName(entity->GetObjectID(), GeneralUtils::UTF8ToUTF16(name), UNASSIGNED_SYSTEM_ADDRESS);
 	}
+
+	void ShowAll(Entity* entity, const SystemAddress& sysAddr, const std::string args) {
+		const auto splitArgs = GeneralUtils::SplitString(args, ' ');
+		CBITSTREAM;
+		BitStreamUtils::WriteHeader(bitStream, eConnectionType::CHAT, eChatMessageType::SHOW_ALL);
+		bitStream.Write(entity->GetObjectID());
+		if (splitArgs.size() > 0 && splitArgs.at(0).size() > 0) bitStream.Write(LUString(splitArgs.at(0), 1));
+		else bitStream.Write(LUString("1", 1));
+		if (splitArgs.size() > 1) bitStream.Write(LUString(splitArgs.at(1), 1));
+		else bitStream.Write(LUString("1", 1));
+		Game::chatServer->Send(&bitStream, SYSTEM_PRIORITY, RELIABLE, 0, Game::chatSysAddr, false);
```

Comment:
move this stream writing and probably send to a dNet ChatPackets function

Reply:
having issues moving these to chatpackets, will discuss more in discord

Reply:
due to the linker issue with chat server, compromised and made (de)serialize functions to contain the data and reading/writing